src/test/java/com/iota/iri/controllers/TransactionViewModelTest.java

```diff
@@ -438,11 +438,19 @@ public void updateTxShouldBeSavedBeforeRelease() throws Exception {
             tvms[i].store(tangle, snapshot);
         }
         tvms[0].isMilestone(tangle, snapshot, true);
-        tvms[1].isMilestone(tangle, snapshot, true);
+        tvms[1].updateSolid(tangle, snapshot, true);
+        tvms[2].updateSender(tangle, snapshot, "sender");
+        tvms[0].isMilestone(tangle, snapshot, false); //Another update to kick the previous tx from cache
 
-        Hash hash = tvms[0].getHash();
-        TransactionViewModel tvm = new TransactionViewModel((Transaction) tangle.load(Transaction.class, hash), hash);
-        Assert.assertTrue("TVM should be a milestone", tvm.isMilestone());
+        Hash hash0 = tvms[0].getHash();
+        TransactionViewModel tvm0 = new TransactionViewModel((Transaction) tangle.load(Transaction.class, hash0), hash0);
+        Assert.assertTrue("TVM should be a milestone", tvm0.isMilestone());
+        Hash hash1 = tvms[1].getHash();
+        TransactionViewModel tvm1 = new TransactionViewModel((Transaction) tangle.load(Transaction.class, hash1), hash1);
+        Assert.assertTrue("TVM should be solid", tvm1.isSolid());
+        Hash hash2 = tvms[2].getHash();
+        TransactionViewModel tvm2 = new TransactionViewModel((Transaction) tangle.load(Transaction.class, hash2), hash2);
+        Assert.assertEquals("TVM sender should be equal", "sender", tvm2.getSender());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=5333738)